src/gs-updates-page.c

```diff
@@ -646,6 +650,21 @@ _get_app_sort_key (GsApp *app)
 	 * 4. online device firmware */
 	g_string_append_printf (key, "%u:", _get_app_section (app));
 
+	switch (gs_app_get_state (app)) {
+	case AS_APP_STATE_INSTALLING:
+		g_string_append (key, "1:");
+		break;
+	case AS_APP_STATE_QUEUED_FOR_INSTALL:
+		g_string_append (key, "2:");
+		break;
+	case AS_APP_STATE_REMOVING:
```

Comment:
Not sure if this case is relevant.

Reply:
If it's being removed, then maybe it shouldn't be shown in the updates page?
In any case, ignore the removing and queued for install, since apps in these states should not be shown in the updates page.